rickshaw-run

```diff
@@ -1667,6 +1667,7 @@ sub process_cmdline() {
             $run{$p} = $defaults{$p};
         }
     }
+    my $base_rb_leader_cmd = "--role=leader --redis-server=localhost --redis-password=" . $run{'roadblock-password'};
```

Comment:
This doesn't appear to be working and I'm wondering if the usage of `my` here is the reason why.  Does this mess with the variable scope?

@atheurer thoughts?

Reply:
> Ok I think this should be fixed with [this commit](https://github.com/perftool-incubator/rickshaw/pull/607/commits/0b9981732b5f3a3f218cd8221681b8027c368db9). Essentially just remove the `my` keywork. This should now reassign `base_rb_leader_cmd`, instead of creating a new local copy in that scope. As you can tell, I'm a perl novice

I think this is a pretty easy mistake to make.  I'm certainly no perl ninja but I've written a fair amount and I wasn't 100% sure that this was the problem until I did my example I just posted.